core/trino-main/src/test/java/io/trino/sql/planner/TestInsert.java

```diff
@@ -24,6 +24,7 @@
 import io.trino.plugin.tpch.TpchPartitioningHandle;
```

Comment:
Please improve the commit message:
```
Test insertwith row filter with missing columns
```

Please also improve PR title too.

Also please add a test to `io.trino.sql.query.TestRowFilter` instead.